processing/src/main/java/io/druid/segment/filter/AndFilter.java

```diff
@@ -42,10 +43,9 @@
 
   private final List<Filter> filters;
 
-  public AndFilter(
-      List<Filter> filters
-  )
+  public AndFilter(List<Filter> filters)
   {
+    Preconditions.checkArgument(filters.size() > 0, "Can't construct empty AndFilter");
```

Comment:
This is a change in behavior, old behavior allowed filter size of 0 and returned `false`